tests/token_bridge_test.cairo

```diff
@@ -339,3 +338,116 @@ fn disable_withdrwal_not_owner() {
     );
 }
 
+#[test]
+fn unblock_token_ok() {
+    let mut mock = mock_state_testing();
+    let usdc_address = deploy_erc20("usdc", "usdc");
```

Comment:
do we need to deploy a token?

Reply:
Oh ya makes sense, can mock an address to do so as well

Reply:
yep, makes sense to do it here

Reply:
Updated ✅